lib/index.coffee

```diff
@@ -68,6 +101,10 @@ class DockerGC
 					else
 						throw new Error('Could not detect architecture of remote host')
 
+	setMetricsFor: (metricsObj, host) ->
```

Comment:
it seems this is never called

Reply:
This is called from the Builder, which injects the metrics object. The setup is quite special due to this being legacy code: the builder sets up the metrics endpoint and adds its own metrics, so, in order not to avoid having a second metrics endpoint, I made the Builder inject its metrics object to each of the GC objects (we have 3 objects: 1 for x86 builds, and 2 for ARM builds)